ultravox/training/config_base.py

```diff
@@ -3,14 +3,32 @@
 import logging
 import os
 from pathlib import Path
-from typing import List, Optional
+from typing import Any, Dict, List, Optional
 
 import simple_parsing
 import torch
+from pydantic import BaseModel
 
 from ultravox.model import ultravox_config
 
 
+class DataDictConfig(BaseModel):
```

Comment:
I think we might want this to live in ultravox/data so that we can integrate it into our other tooling, and also write tests for it.